src/workers/personalizedDigestEmail.ts

```diff
@@ -43,6 +52,14 @@ const setEmailSendDate = async ({
   );
 };
 
+const sendTypeToFeatureMap: Record<
+  UserPersonalizedDigestSendType,
+  Feature<PersonalizedDigestFeatureConfig>
+> = {
+  [UserPersonalizedDigestSendType.weekly]: features.personalizedDigest,
+  [UserPersonalizedDigestSendType.workdays]: features.dailyDigest,
+};
+
```

Comment:
Basically we have to do this matching since in RFC we opted to save send state in our db sp we cant just rely on feature flag to know which config to load. This will do for initial daily digest test but if we introduce more send types we will have to move to just evaluating GB feature in cron and then pass the value of digest config to worker directly.